main.go

```diff
@@ -26,7 +26,10 @@ func main() {
 
 	api := slack.New(cnf.SlackToken)
 
-	c, err := selectChannel(coll, api)
+	var ignore Channels
+	ignore = append(ignore, Channel{Name: cnf.DestChan})
```

Comment:
There's not a real good reason to break it down farther than `ignore := Channels{Channel{Name: cnf.DestChan}}`, is there? Going more makes the code smaller, but I feel like this point is the most easily readable for anyone not too familiar with the project. It clearly states what all is going on.